examples/flask-hello-world-pyproject/pyproject.toml

```diff
@@ -2,16 +2,21 @@
 name = "flask-hello-world-pyproject"
 version = "0.1.0"
 description = "Example how to get started with flask in a pixi environment."
-license = "MIT OR Apache-2.0"
-homepage = "https://github.com/prefix/pixi"
 readme = "README.md"
 requires-python = ">=3.11"
 dependencies = ["flask==2.*"]
 
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
 [tool.pixi.project]
 name = "flask-hello-world-pyproject"
 channels = ["conda-forge"]
-platforms = ["linux-64"]
+platforms = ["linux-64", "osx-arm64"]
+
+[tool.pixi.pypi-dependencies]
+flask-hello-world-pyproject = { path = ".", editable = true }
 
 [tool.pixi.tasks]
-start = "python -m flask run --port=5050"
+start = "python -m flask --app flask_hello_world_pyproject.app:app run --port=5050"
```

Comment:
The integrated tests test the `test` command. Maybe we can use this to test if the install went correctly? No really a good way to test if it's editable, but we can do that seperately with the lock file.

Reply:
Done